### ISSUE TITLE
Upgrade libraries

### ISSUE THREAD

**Issue by stefan-it** [dbmdz/blueprints#35]
After #34 several libraries, like `dc-commons-springboot` or `git-commit-id-plugin` should be updated to their latest version.

**Comment by morpheus-87** [comment]
Already done, right @stefan-it?

**Comment by datazuul** [comment]
job of dependabot if not done so far...
